cli/config/configuration-qemu.toml.in

```diff
@@ -218,6 +218,13 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # Default false
 #hotplug_vfio_on_root_bus = true
 
+# Before hot plugging a PCIe device, you need to add a pcie_root_port device.
+# Use this parameter when using some large PCI bar devices, such as Nvidia GPU
+# The value means the number of pcie_root_port
+# This value is valid for "q35" machine type
+# Default 1
+pcie_root_port = 1
```

Comment:
by default this should be 0, since not all users need a pcie root port

Reply:
OK